src/detection_win.cpp

```diff
@@ -89,6 +89,7 @@ _SetupDiGetDeviceRegistryProperty DllSetupDiGetDeviceRegistryProperty;
  * Local Helper Functions protoypes
  **********************************/
 void UpdateDevice(PDEV_BROADCAST_DEVICEINTERFACE pDevInf, WPARAM wParam, DeviceState_t state);
+std::string Utf8Encode(const std::string &str);
```

Comment:
Is `std::string` still available after https://github.com/MadLittleMods/node-usb-detection/pull/84 is merged?

Reply:
No problem, 'std::string' is the c++ standard，mean string with namespace std,
#84 problem is the 'ATL' is not the standard library.